### ISSUE TITLE
Threads breaking library import

### ISSUE THREAD

**Issue by scadergit** [codesandbox/codesandbox-client#6415]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the [Contributing Guidelines][contributing] for this project.
- [x] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem
When the code attempts to compile after importing a library, it breaks when reading a thread's registerSerializer property.
![image](https://user-images.githubusercontent.com/17500494/150610982-7f490324-007f-435c-a93e-fe78a44e1b14.png)

## How has this issue affected you? What are you trying to accomplish?
Cannot import library, which imports fine locally and elsewhere.

### To Reproduce
Create a new sandbox based on Vue, then add VueLayers (vuelayers) as dependency and import it. 

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | 918060405 (?)  |
| Browser          |  Chrome |
| Operating System | Windows 10 |

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by scadergit** [comment]
Thanks, @JamesACS ...will there be updates on this thread to know when I can use it?

**Comment by JamesACS** [comment]
Yep @scadergit - When there are any updates I'll share them here.
